components/server/src/authorization/spicedb-authorizer.ts

```diff
@@ -110,7 +111,7 @@ export class SpiceDBAuthorizer {
                 log.error("[spicedb] Failed to perform authorization check.", err, {
                     request: new TrustedValue(req),
                 });
-                return { permitted: !featureEnabled };
+                throw new ApplicationError(ErrorCodes.INTERNAL_SERVER_ERROR, "Failed to perform authorization check.");
```

Comment:
If `!featureEnabled` we should return `permitted = true` and only throw if the feature is enabled

Reply:
makes sense, but from code it looks like already a case, since we don't await?